core/CMakeLists.txt

```diff
@@ -18,6 +18,15 @@ set_target_properties(core PROPERTIES
 	VERSION "${PROJECT_VERSION}"
 	NO_SONAME ON
 )
+if(APPLE)
+	set_target_properties(core PROPERTIES
+		POSITION_INDEPENDENT_CODE ON
+		MACOSX_RPATH ON
+		LINKER_LANGUAGE "CXX"
+		VERSION "${PROJECT_VERSION}"
+		NO_SONAME OFF
+	)
+endif()
```

Comment:
Do you need to copy the full block? Why not just `set_target_property` the two deviating properties?

Reply:
Resolved.